db/migrate/20160719170253_add_biography_to_profile.rb

```diff
@@ -1,5 +1,5 @@
 class AddBiographyToProfile < ActiveRecord::Migration
   def change
-    add_column :profiles, :biography, :text, default: ""
+    add_column :profiles, :biography, :text
```

Comment:
Ahh. You can't edit a migration once it's been migrated! You'd need to create a new migration that uses `change_column`. I suggest that you `git revert` this to avoid all sorts of issues